template/package.json

```diff
@@ -7,17 +7,23 @@
   "main": "index.js",
   "scripts": {
     "dev": "node serve-local.js",
-    "deploy-dev": "nuxt build && serverless deploy --stage dev",
-    "deploy-dev-fast": "nuxt build && serverless deploy --stage dev --function main",
-    "deploy-prod": "nuxt build && serverless deploy --stage prod"
+    "build-nuxt-es5": "node scripts/build-nuxt-es5",
+    "build": "npm run build-nuxt-es5 && cross-env AWS_EXECUTION_ENV=nodejs6.10 node -r ./src-nuxt/lib/node-8-backcompat -require ./src-nuxt/lib/babel-register node_modules/.bin/nuxt build",
```

Comment:
`cross-env` should be then listen under devDependencies ☝️

Reply:
Done in f0d0d4c

Reply:
I got `bad options: -require` error when I run `deploy-dev`. Switching to `-r` works. Maybe because I'm using node 8.9, but when I use node 6.10.3 I can't run the app locally.

Reply:
Apologies for the typo. If a dev runs the app locally it is assumed that the dev is aware of the nuances involved and would specify `AWS_EXECUTION_ENV` so the transpiled nuxt is picked up.